src/js/grapl-cdk/lib/redis.ts

```diff
@@ -0,0 +1,46 @@
+import * as cdk from "@aws-cdk/core";
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as elasticache from "@aws-cdk/aws-elasticache";
+
+export class RedisCluster extends cdk.Construct {
+    readonly securityGroup: ec2.SecurityGroup;
+    readonly connections: ec2.Connections;
+    readonly cluster: elasticache.CfnCacheCluster;
+
+    constructor(scope: cdk.Construct, id: string, vpc: ec2.IVpc) {
+        super(scope, id);
+
+        // Define a group for telling Elasticache which subnets to put cache nodes in.
+        const subnetGroup = new elasticache.CfnSubnetGroup(this, id + '-subnet-group', {
+            description: `List of subnets used for redis cache ${id}`,
+            subnetIds: vpc.privateSubnets.map(function(subnet) {
```

Comment:
`vpc.privateSubnets.map(subnet => subnet.subnetId),`

I don't really care if we use this, but just fwiw since we're all learning js/ts we can use an arrow function here to cut it down.